Backpack-SwiftUI/Card/Classes/BPKDividedCard.swift

```diff
@@ -0,0 +1,88 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright © 2022 Skyscanner Ltd. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+public struct BPKDividedCard<PrimaryContent, SecondaryContent>: View where PrimaryContent: View, SecondaryContent: View {
+    private var isPrimaryContentPadded: Bool
+    private var isSecondaryContentPadded: Bool
+    private let primaryContent: PrimaryContent
+    private let secondaryContent: SecondaryContent
+    private var tapAction : () -> Void = {}
+
+    public init(
+        isPrimaryContentPadded: Bool = true,
+        isSecondaryContentPadded: Bool = true,
```

Comment:
We decided to not have a padded property after all. The card will have no padding by default and the consumer will need to add padding to their own subviews. Would you mind changing the component to match that decision?

Reply:
> We decided to not have a padded property after all. The card will have no padding by default and the consumer will need to add padding to their own subviews. Would you mind changing the component to match that decision?

I will change the component. It's better to add padding to subviews.😄